detector/filecontent/shannon_entropy.go

```diff
@@ -1,8 +1,8 @@
-package detector
```

Comment:
This  Entropy object independent of any detector or file-content. It is just a utility that takes a string , a base character set and calculates Shannon entropy. Perhaps this can go into a utility package ?

The same argument can be made about base64 and hex detectors.
They essentially process bytes/strings.

Reply:
ShannonEntropy, base64 and hex all are three are used only by filecontent_detector. So I moved to filecontent folder. 

If we are using across detectors, may be we can move to utility.

Thoughts ?

Reply:
You are right. This logic can be extended to all three types of detectors. Perhaps this can be handled as part of a separate refactoring.